statsmodels/discrete/discrete_model.py

```diff
@@ -400,8 +400,8 @@ def cov_params_func_l1(self, likelihood_model, xopt, retvals):
         """
         H = likelihood_model.hessian(xopt)
         trimmed = retvals['trimmed']
-        nz_idx = np.nonzero(trimmed == False)[0]
```

Comment:
If these are bool arrays then this comparrison is more meaningful than comparing to int 0, and IMO should remain. Same goes for other numpy usage.  This is a bad pep8 rule probably, at least in some application.

Reply:
If these are bool arrays, you can just do np.nonzero(~trimmed)